raystack/frontier/v1beta1/admin.proto

```diff
@@ -814,4 +826,14 @@ message UpdateBillingAccountLimitsRequest {
   int64 credit_min = 3;
 }
 
-message UpdateBillingAccountLimitsResponse {}
\ No newline at end of file
+message UpdateBillingAccountLimitsResponse {}
+
+message SetOrganizationKycRequest {
+  string organization_id = 1;
+  bool status = 2;
```

Comment:
`status` and `link` should be present inside `body` key and `body: "body"` should be passed in rpc.

Reply:
`body: "*"` will make the whole message as body. then it will duplicate the org_id in both body and path

Reply:
yes, except the one present in the path: https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L144

This is how the swagger looks: 
<img width="950" alt="Screenshot 2025-02-19 at 1 00 31 PM" src="https://github.com/user-attachments/assets/68c9c70a-dd72-4f26-8ea2-77bee2f263c9" />

It does not duplicate the org_id in the body. It only appears in the path.

Reply:
Okay, my bad.